CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # TAMR AWS EMR module
 
+## v4.0.0 - June 15, 2021
+* Update cluster to use instance fleet, for a mix of on-demand and spot instances
```

Comment:
Be specific about which variables have been removed or added.

I know that we're adding a lot of variables, so maybe we don't have to be exhaustive, but we should cover the ones that someone will need to update on upgrade. So that will be all removed and what they were replaced with (e.g.   core_group_instance_count   -> core_instance_on_demand_count).

This is helpful for someone upgrading from the previous version to know what needs to change.